run/rails/Dockerfile

```diff
@@ -2,7 +2,9 @@
 # https://hub.docker.com/_/ruby
 FROM ruby:3.0
```

Comment:
The new node command assumes the "buster" version of the debian OS, so we should probably lock the base image to buster as well, and maybe add a comment to that effect:

```
# Pinning the OS to buster because the nodejs install script is buster-specific.
# Be sure to update the nodejs install command if the base image OS is updated.
FROM ruby:3.0-buster
```

Reply:
That's good to know! Thanks Daniel. I just updated the Dockerfile to pin the ruby base image to buster